src/components/Modal.tsx

```
@@ -1,10 +1,18 @@
-import { Box, Modal, Typography } from '@mui/material';
+import { Box, Divider, Modal, Typography } from '@mui/material';
 
 import CloseSharpIcon from '@mui/icons-material/CloseSharp';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 import ButtonFunctional from './ButtonComponent';
 import { modalStyles } from '@/styles/modalStyles';
+import { useTheme } from '@mui/material/styles';
+
+import {
+  deepClone,
```

ESLint

[error] 'deepClone' is defined but never used.

(no-unused-vars)